ssoadmin/client.go

```diff
@@ -160,6 +160,39 @@ func (c *Client) DeletePrincipal(ctx context.Context, name string) error {
 	return err
 }
 
+func (c *Client) AddUsersToGroup(ctx context.Context, groupName string, userIDs []types.PrincipalId) error {
```

Comment:
How about we make this a variadic param, `userIDs ...types.PrincipalId`?  That way it can be called with 1 or more IDs and the `AddUserToGroup` method below can be removed.

Reply:
That's great!